sleap/instance.py

```diff
@@ -1141,6 +1141,19 @@ def __delitem__(self, index):
         # Modify the instance to remove reference to this frame
         value.frame = None
 
+    def __repr__(self) -> str:
+        """Return a readable representation of the LabeledFrame."""
+        return (
+            f"LabeledFrame(video={type(self.video.backend).__name__}"
+            f"('{self.video.filename}'), "
+            f"frame_idx={self.frame_idx}, "
+            f"instances={len(self.instances)})"
+        )
+
+    def __str__(self) -> str:
```

Comment:
This might be redundant (I think str defaults to repr when not implemented)

Reply:
Ack -- this was because I was overriding the default attrs behavior which creates a method for each. Just turning it off with the attrs decorator now.